src/assets/icons/DarkTheme/index.js

```diff
@@ -0,0 +1,126 @@
+            fill="white"
+          />
+          <rect
+            x="44.8226"
+            y="26.9216"
+            width="0.701376"
+            height="3.32196"
+            transform="rotate(-90 44.8226 26.9216)"
+            fill="white"
+          />
+          <rect
+            x="47.1954"
+            y="23.8823"
+            width="0.711863"
+            height="3.27308"
+            fill="white"
+          />
+        </g>
+        <rect
+          x="16.5698"
+          y="58.5322"
+          width="49.0465"
+          height="13.9679"
+          rx="5"
+          fill="white"
+        />
+        <rect
+          x="16.5698"
+          y="78.4863"
+          width="33.8023"
+          height="6.65137"
+          rx="3.32569"
+          fill="white"
+          fillOpacity="0.5"
+        />
+        <rect
+          x="16.5698"
+          y="88.4634"
+          width="26.5116"
+          height="6.65137"
+          rx="3.32569"
+          fill="white"
+          fillOpacity="0.5"
+        />
+        <rect
+          x="16.5698"
+          y="98.4404"
+          width="29.1628"
+          height="6.65137"
+          rx="3.32569"
+          fill="white"
+          fillOpacity="0.5"
+        />
+        <path
+          d="M0 0V-3H-3V0H0ZM228 0H231V-3H228V0ZM228 145V148H231V145H228ZM0 145H-3V148H0V145ZM0 3H228V-3H0V3ZM225 0V145H231V0H225ZM228 142H0V148H228V142ZM3 145V0H-3V145H3Z"
+          fill={theme['main-font']}
+        />
+        <defs>
+          <filter
+            id="filter0_d"
+            x="16.5698"
+            y="19.2891"
+            width="57.0465"
+            height="21.3029"
+            filterUnits="userSpaceOnUse"
+            colorInterpolationFilters="sRGB"
+          >
+            <feFlood floodOpacity="0" result="BackgroundImageFix" />
+            <feColorMatrix
+              in="SourceAlpha"
+              type="matrix"
+              values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+              result="hardAlpha"
+            />
+            <feOffset dx="4" dy="4" />
+            <feGaussianBlur stdDeviation="2" />
+            <feColorMatrix
+              type="matrix"
+              values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"
+            />
+            <feBlend
+              mode="normal"
+              in2="BackgroundImageFix"
+              result="effect1_dropShadow"
+            />
+            <feBlend
+              mode="normal"
+              in="SourceGraphic"
+              in2="effect1_dropShadow"
+              result="shape"
+            />
+          </filter>
+        </defs>
+      </svg>
```

Comment:
Isso aqui ta bem grande e bem dificil de ler. Tem alguma forma de simplificar isso? (maps, constantes, comentários e etc)

Reply:
Analisando aqui uma forma de fazer isso com Map e tals, eu acho que usar maps nesse caso tornaria o código ainda mais confuso na verdade. Eu penso que como é um ícone svg talvez deixar assim seja a melhor forma :/

Reply:
Acho que uma forma possivel de refatorar esse arquivo é colocando comentários nele em cima (`// Dark Theme icon for user selection`) e mudando o nome do componente para `DarkThemeIcon`.

Reply:
mudei :)